2024/json-ld-wg/index.html

```diff
@@ -178,39 +179,62 @@ <h2>Motivation and Background</h2>
-          processing model, and specifying improved context retrieval methods for greater clarity and stability.
+          The goal of this new, more inclusively focused JSON-LD WG is to meet the demand of the broader
+          JSON-LD community by strengthening the foundational JSON-LD specifications, growing the family 
+          of JSON-LD related formats which orbit around the JSON-LD processing model, and specifying improved
+          context retrieval methods for greater clarity and stability.
         </p>
+
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
 				<p>
-					The Working Group will extend the recommendations defining the JSON-LD specifications (i.e., <a href="https://www.w3.org/TR/json-ld11/">JSON-LD 1.1</a>, <a href="https://www.w3.org/TR/json-ld11-api/">JSON-LD 1.1 API</a>, <a href="https://www.w3.org/TR/json-ld11-framing/">JSON-LD 1.1 Framing</a>)
-          with the features introduced by <a href="https://www.w3.org/TR/rdf12-concepts">RDF 1.2</a>
-          in addition to open Errata and requested features
-          as noted on the groups <a href="https://github.com/orgs/w3c/projects/84">Project Management Page</a>.
-          Together, these specifications provide a JSON format for Linked Data,
-          enabling interoperation in a way which is familiar to and usable by software engineers.
-					<!-- Note that the JSON-LD APIs are not browser specific; while appropriate for use within browsers, they are not limited to such use, and there is no requirement for browsers to implement them natively. -->
-				</p>
+					The Working Group will evolve the recommendations defining the JSON-LD specifications 
+          (i.e., <a href="https://www.w3.org/TR/json-ld11/">JSON-LD 1.1</a>, 
+          <a href="https://www.w3.org/TR/json-ld11-api/">JSON-LD 1.1 API</a>, and
+          <a href="https://www.w3.org/TR/json-ld11-framing/">JSON-LD 1.1 Framing</a>) by handling Errata 
+          and by adding requested features as recorded on the group's 
+          <a href="https://github.com/orgs/w3c/projects/84">Project Management Page</a>. In order to handle these, 
+          the Working Group will issue new versions of these recommendations, possibly with new features,
+          in order to render future usage and evolutions easier. In particular, the Working Group will handle 
+          two important security related problems:
+        </p>
+
+
+        <ol>
+          <li>[Placeholder: safe mode]</li>
+          <li>[Placeholder: context file hash, and related stuff]</li>
+        </ol>
+
+        <!-- Note that the JSON-LD APIs are not browser specific; while appropriate for use within browsers, they are not limited to such use, and there is no requirement for browsers to implement them natively. -->
+
+        <p>
+          The development of the 1.2 versions of the JSON-LD specification will follow the principle of
```

Comment:
Are there multiple specifications?
```suggestion
          Development of the 1.2 versions of the JSON-LD specifications will follow the principle of
```

Reply:
or multiple versions?

Reply:
They're more or less a "suite" of specifications. The release of v1.1 included JSON-LD Syntax, API, as well as Framing.

Reply:
> They're more or less a "suite" of specifications

OK. [The first](https://github.com/json-ld/json-ld-wg-charter/pull/23#discussion_r2319709079) of my suggestions above applies.